control/preimage/src/helpers.rs

```diff
@@ -89,8 +168,12 @@ pub async fn query_weight(api: &OnlineClient<PolkadotConfig>, call: BridgeHubRun
     Ok((call_info.weight.ref_time, call_info.weight.proof_size))
 }
 
-pub async fn query_para_id(api: &OnlineClient<PolkadotConfig>) -> Result<u32, Box<dyn std::error::Error>> {
-    let storage_query = bridge_hub_runtime::storage().parachain_info().parachain_id();
+pub async fn query_para_id(
```

Comment:
```suggestion
pub async fn query_para_id_bridge_hub(
```

Reply:
thanks, though this is now obsolete code.